Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/modules/color-picker/ColorContextPad.js

```diff
@@ -51,6 +51,10 @@ export default class ColorContextPad {
   getContextPadEntries(element) {
     const { _canvas, _contextPad, _colorPalette, _modeling } = this;
 
+    if (is(element, 'bpmn:TextAnnotation')) {
+      return;
+    }
```

Comment:
The color pallete icon is still appearing for text annotations and coloring the text a lighter version of the selected color.

Reply:
I think what the ticket meant is the element itself. The line connecting the annotation can still be coloured.